.github/workflows/buildtest.yml

```
@@ -110,3 +110,13 @@ jobs:
           path: |
             junit_test_summary-${{ matrix.test-chunk }}.xml
           retention-days: 1
+  buildtest_summary:
```

```suggestion
  ensure_all_tests_pass:
```
Or something similar. If you call it "summary", I'd assume it is somehow writing the summary somewhere.